onnxruntime/core/platform/windows/debug_alloc.cc

```diff
@@ -210,7 +210,7 @@ Memory_LeakCheck::~Memory_LeakCheck() {
     //     empty_string = new string;
     //     empty_named_groups = new std::map<string, int>;
     //     empty_group_names = new std::map<int, string>; });
-    if (string.find("re2::RE2::Init") == std::string::npos &&
+    if (string.find("RtlRunOnceExecuteOnce") == std::string::npos &&
```

Comment:
It's a Windows Symbol. Are you sure you want to do this?

Reply:
Yes, it is a windows source code :)